buildSrc/build.gradle

```diff
@@ -163,6 +163,10 @@ if (project != rootProject) {
   apply plugin: 'nebula.maven-base-publish'
   apply plugin: 'nebula.maven-scm'
 
+  // we need to apply these again to override the build plugin
+  targetCompatibility = "10"
```

Comment:
Is this a separate change from this? Like, is it just required because of this change?

Reply:
explained below